### ISSUE TITLE
How to resolve a certificate issue

### ISSUE THREAD

**Issue by seamusdemora** [ytdl-org/youtube-dl#25543]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

### REFERENCES: #25521, #25491, #25532, #25539

It's been said that, "I've already [pointed out](https://github.com/ytdl-org/youtube-dl/issues/25491#issuecomment-637047754): yt-dl.org certificate is fine and not expired.". (#25539)

In that ['answer'](https://github.com/ytdl-org/youtube-dl/issues/25491#issuecomment-637047754), a reference to support resolution of the issue is provided at [this page](https://support.sectigo.com/Com_KnowledgeDetailPage?Id=kA03l00000117LT)

That page includes the following passage as a resolution (?) for the issue at hand (#25539):

>What You Need to Do
>For most use cases, including certificates serving modern client or server systems, no action is required, whether or not you have issued certificates cross-chained to the AddTrust root. ...

In summary then: @dstftw  says the cert is fine/not expired, and the CA says I need do nothing. **Yet I still get the expiration warning while following the documented download instructions.**

**Comment by comsomisha** [comment]
Let see https://github.com/ytdl-org/youtube-dl/issues/7309#issuecomment-172354527
You should delete expired certs in your OS.
